app/controllers/v0/sessions_controller.rb

```diff
@@ -93,5 +109,27 @@ def async_create_evss_account(user)
       auth_headers = EVSS::AuthHeaders.new(user).to_h
       EVSS::CreateUserAccountJob.perform_async(auth_headers)
     end
+
+    # :nocov:
+    def handle_completed_slo
+      logout_response = OneLogin::RubySaml::Logoutresponse.new(params[:SAMLResponse], saml_settings)
+
+      logger.info "LogoutResponse is: #{logout_response}"
+
+      if !logout_response.validate
+        logger.error 'The SAML Logout Response is invalid'
```

Comment:
We'd also get a validation failure if somebody sent us an incorrectly-signed request. And I don't think we'd get anything back if the user failed authentication.